### ISSUE TITLE
Add testing files for the pipeline

### ISSUE THREAD

**Issue by bhillmann** [knights-lab/shi7#14]
Currently, we have no defined way to test the software. This is bad practice.

**Comment by GabeAl** [comment]
We now have some test cases scattered around, but it's not in the repo (so yes, bad practice indeed). 

@kaiweiang , would you mind uploading all the test files and commandlines you've accumulated over time?

**Comment by bhillmann** [comment]
I was thinking to eventually add unit test cases. Any suggestions for this software? I know there are several options in Python and we could emulate open source best practices. Any interest in learning a continuous integration suite?

For now, it might be simplest to include some small FASTQ files to the repository to check the pipeline.

**Comment by GabeAl** [comment]
Good idea. I think this is fairly essential as we expand the number of possible run configurations. The more combinations of options possible, the more likely something subtle will go wrong, and unit testing becomes more relevant. Yes. 

For now (1.0), some test files are probably enough, and we can verify they're working with the common commandlines.

**Comment by GabeAl** [comment]
Added test files. Still need tests for gotta_split though!

**Comment by bhillmann** [comment]
Preliminary testing files have been added for other files. @GabeAl Can you make the gotta_split ones?
